lib/opensusebasetest.pm

```diff
@@ -742,7 +742,8 @@ sub wait_grub_to_boot_on_local_disk {
         }
     }
 
-    assert_screen(\@tags, 15);
+    # We need to wait more time for aarch64's tianocore-mainmenu
+    (check_var('ARCH', 'aarch64')) ? assert_screen(\@tags, 30) : assert_screen(\@tags, 15);
```

Comment:
you can simplify to: `assert_screen(\@tags, check_var('ARCH', 'aarch64') ? 30 : 15)`